Azure-Samples/java-on-azure-examples
# Improve names in pom.xml

## Purpose

* Fixes #149

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Doing a Maven build from the root you will see the new names
* In an IDE such as Intellij IDEA, the Maven window with all the Maven projects displays the names
